app/lib/utils/btc.js

```diff
@@ -18,7 +25,7 @@ export function btcToBits(btc) {
 
 export function btcToUsd(btc, price) {
   const amount = parseFloat(btc * price).toFixed(2)
-  return (btc > 0 && amount <= 0) ? '< 0.01' : amount.toLocaleString('en')
+  return (btc > 0 && amount <= 0) ? '< 0.01' : numberWithCommas(amount)
```

Comment:
Rather than calling this, could we use the built in currency support of  `Number.prototype.toLocaleString()` to get us an appropriately formatted amount.

See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString#Parameters